subsystems/timekeeper.py

```diff
@@ -0,0 +1,26 @@
+import time
+import wpilib
+
+#Teleop duration
+TELEOP_DURATION_SECONDS = 150
+
+
+class MatchTimer:
```

Comment:
I think you need to work on this more.     What do you want this thing to do? and what methods should it have.

For example, if we want to do a countdown from 10 down to 1,  then it should have a simple call for something that tells if this call is a countdown event.     It could return the number if we are at the second of the countdown, and it could otherwise return None.    This permits our display client to send the appropriate updates.

tminus_seconds = self.timeKeeper.secondsRemaining(10)       #  ask if there are 10 or fewer seconds remaining
if tminus_seconds is not None                                                      #  None if too early or between seconds
    self.displayClient.sendCountdownDigit(tminus_seconds)

So, inside timekeeper, you need to keep track of when you discover 10 seconds remaining
and return the seconds on the second.